src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -148,11 +167,12 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Xml" />
     <Reference Include="WindowsBase" />
-	<PackageReference Include="Extended.Wpf.Toolkit" version="3.0.0">
+	<PackageReference Include="Extended.Wpf.Toolkit" version="4.5.0">
```

Comment:
yeah, this is a good unblocker, but like you said we can't ship this.

Reply:
Ok, I returned to 3.0.0

Reply:
@marbelarrietaGlobant - sorry I was not clear, this comment was more intended to just be a reminder to others on the team that we have work to do here.

Does this actually still build if we reference 3.0 somehow?

Reply:
Well, yes! the project with 3.0.0 still builds with no build errors but a warning:
 warning NU1701:  Package 'Extended.Wpf.Toolkit 3.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net6.0-windows7.0'. This package may not be fully compatible with your project.